boards/riscv/tlsr9518adk80d/pinmux.c

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2021 Telink Semiconductor
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gpio.h>
+#include <drivers/pinmux.h>
+
+
+/**
+ *  @brief  Define UART0 pins : TX(PA3 PB2 PD2), RX(PA4 PB3 PD3)
+ */
+#define UART0_TX_PA3            GPIO_PA3
+#define UART0_TX_PA3_FUNC       PINMUX_FUNC_B
+
+#define UART0_TX_PB2            GPIO_PB2
+#define UART0_TX_PB2_FUNC       PINMUX_FUNC_C
+
+#define UART0_TX_PD2            GPIO_PD2
+#define UART0_TX_PD2_FUNC       PINMUX_FUNC_A
+
+
+#define UART0_RX_PA4            GPIO_PA4
+#define UART0_RX_PA4_FUNC       PINMUX_FUNC_B
+
+#define UART0_RX_PB3            GPIO_PB3
+#define UART0_RX_PB3_FUNC       PINMUX_FUNC_C
+
+#define UART0_RX_PD3            GPIO_PD3
+#define UART0_RX_PD3_FUNC       PINMUX_FUNC_A
+
+
+/**
+ *  @brief  Define UART1 pins: TX(PC6 PD6 PE0), RX(PC7 PD7 PE2)
+ */
+#define UART1_TX_PC6            GPIO_PC6
+#define UART1_TX_PC6_FUNC       PINMUX_FUNC_C
+
+#define UART1_TX_PD6            GPIO_PD6
+#define UART1_TX_PD6_FUNC       PINMUX_FUNC_A
+
+#define UART1_TX_PE0            GPIO_PE0
+#define UART1_TX_PE0_FUNC       PINMUX_FUNC_B
+
+
+#define UART1_RX_PC7            GPIO_PC7
+#define UART1_RX_PC7_FUNC       PINMUX_FUNC_C
+
+#define UART1_RX_PD7            GPIO_PD7
+#define UART1_RX_PD7_FUNC       PINMUX_FUNC_A
+
+#define UART1_RX_PE2            GPIO_PE2
+#define UART1_RX_PE2_FUNC       PINMUX_FUNC_B
+
+
+/**
+ *  @brief  Config board pins
+ */
+static int b91_pinmux_init(const struct device *dev)
+{
+	ARG_UNUSED(dev);
+
+	const struct device *pinmux =
+		device_get_binding(DT_LABEL(DT_NODELABEL(pinmux)));
+
+	/* select func2_sub_select */
+	reg_gpio_pad_mul_sel |= BIT(0);
+
+	/* UART0 RX, TX */
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(uart0), okay) && CONFIG_UART_TELINK_B91
+	pinmux_pin_set(pinmux, UART0_TX_PB2, UART0_TX_PB2_FUNC);
+	pinmux_pin_set(pinmux, UART0_RX_PB3, UART0_RX_PB3_FUNC);
+#endif
```

Comment:
Not sure, but @galak may want this in DTS instead.

Reply:
Yes, @galak already asked me to move it to dts during an another PR.
Done.